### ISSUE TITLE
CSS doesn't always load with docs

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/CWRC-Writer#495]
I have gotten this error more than once, with the biography document template and the sample playwrights entry.

![image](https://user-images.githubusercontent.com/545952/34285253-543abbbc-e6a6-11e7-88e2-8865206adcac.png)

**Comment by jchartrand** [comment]
Is this happening on the 'dev' version, i.e., http://208.75.74.225?  I think it is caused by requesting an http url (the css url above) from a page that was loaded using https.  The dev version now has https enabled.  Andrew @ajmacdonald, I think you updated these urls (in the code) to use https - is it just that we now need a new build to dev (and to prod so I can switch it over to https as well)?

**Comment by ajmacdonald** [comment]
https://github.com/cwrc/CWRC-Writer/issues/496#issuecomment-356339273

**Comment by ilovan** [comment]
@SusanBrown , are you getting this error message with the Orlando bio docs as well? You shouldn't since the Orlando stylesheets have not been moved to a different folder (see https://github.com/cwrc/CWRC-Writer/issues/496#issuecomment-356405815)

**Comment by SusanBrown** [comment]
Yes apparently I got it with the bio docs template, as well. It is not consitent.
